static/app/views/teamInsights/teamIssuesReviewed.tsx

```diff
@@ -0,0 +1,206 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import isEqual from 'lodash/isEqual';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import BarChart from 'app/components/charts/barChart';
+import {DateTimeObject} from 'app/components/charts/utils';
+import IdBadge from 'app/components/idBadge';
+import {getParams} from 'app/components/organizations/globalSelectionHeader/getParams';
+import PanelTable from 'app/components/panels/panelTable';
+import Placeholder from 'app/components/placeholder';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization, Project} from 'app/types';
+import {SeriesDataUnit} from 'app/types/echarts';
+import {formatPercentage} from 'app/utils/formatters';
+
+import {convertDaySeriesToWeeks} from './utils';
+
+type IssuesBreakdown = Record<string, Record<string, {reviewed: number; total: number}>>;
+
+type Props = AsyncComponent['props'] & {
+  organization: Organization;
+  projects: Project[];
+  teamSlug: string;
+} & DateTimeObject;
+
+type State = AsyncComponent['state'] & {
+  issuesBreakdown: IssuesBreakdown | null;
+};
+
+function convertKeyValueToSeries(data: Record<string, number>): SeriesDataUnit[] {
+  return Object.entries(data).map(([bucket, count]) => ({
+    value: count,
+    name: new Date(bucket).getTime(),
+  }));
+}
```

Comment:
Is this something that could be moved to `static/app/views/teamInsights/utils.tsx` and reused in `static/app/views/teamInsights/teamAlertsTriggered.tsx`?